pom.xml

```diff
@@ -137,27 +138,26 @@
         <type>pom</type>
         <scope>test</scope>
       </dependency>
-      <!--If not specified, the 2.7.6 spring-boot-starter-parent dependency will introduce transitively the 1.30 version of snakeyaml, for which a multitude of security concerns are reported -->
-      <dependency>
-        <groupId>org.yaml</groupId>
-        <artifactId>snakeyaml</artifactId>
-        <version>2.0</version>
-      </dependency>
```

Comment:
Why was this removed? 
If the version is not specified then sprnig-boot will use the 1.33 version for which security issues have been reported:
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-1471

Reply:
>2.7.6 spring-boot-starter-parent

Reply:
we are now using 3.x

Reply:
I was wrong, turns out it is still pulled but now with version 1.33 (which has 1 CVE)